tests/storage/test_extraction.py

```diff
@@ -70,11 +87,54 @@ def _build_ert(self, path):
         # Default
         f.write(
             "JOBNAME poly_%d\n"
+            "QUEUE_SYSTEM LOCAL\n"
+            "QUEUE_OPTION LOCAL MAX_RUNNING 50\n"
             f"NUM_REALIZATIONS {self.ensemble_size}\n"
-            "INSTALL_JOB job JOB\n"
-            "SIMULATION_JOB job\n"
         )
 
+    def _build_job(self, path):
+        f = (path / "test.ert").open("a")
+        f.write("INSTALL_JOB job JOB\n" "SIMULATION_JOB job\n")
+
+        if self._job_script is None:
+            (path / "JOB").write_text("EXECUTABLE /usr/bin/true\n")
+        else:
+            (path / "JOB").write_text(f"EXECUTABLE {path}/script\n")
+            (path / "script").write_text(self._job_script)
```

Comment:
I think you need `(path / "script").chmod(0o755)`? Or is ERT somehow able to execute it without permissions?